eradiate/scenes/biosphere/tests/test_biosphere_discrete.py

```diff
@@ -236,6 +299,42 @@ def test_discrete_canopy_advanced(mode_mono, tempfile_spheres, tempfile_leaves):
                     filename=tempfile_leaves, id="leaf_cloud_precomputed"
                 ),
             ),
+            InstancedCanopyElement(
+                instance_positions=[[0, 0, 0]],
+                canopy_element=AbstractTree(
+                    leaf_cloud=LeafCloud.from_dict(
+                        {
+                            "construct": "cuboid",
+                            "n_leaves": 100,
+                            "l_horizontal": 10.0,
+                            "l_vertical": 1.0,
+                            "leaf_radius": 10.0,
+                            "leaf_radius_units": "cm",
+                        }
```

Comment:
That's not exactly what "use the regular Python API" implies ;) Would you mind using only Python calls here, so that the intent of this section is consistent with the test code?